### PR TITLE
Add primary Performance Monitoring page

### PR DESCRIPTION
**Pull request by PeloWriter**
Performance Monitoring currently lacks a primary page to guide readers to the adjacent content - this PR adds this content. 

Note that the two SDKs that support performance monitoring - JS and Python - are each listed separately. This will be created as a toggle/selector as the page iterates.

### CODE REVIEW

File: src/collections/_documentation/performance/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+npm install --save @sentry/tracing
+```
+
+Next, initialize the integration in your call to `Sentry.init`:
+
+```jsx
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/tracing';
+Sentry.init({
+  dsn: '<https://bc367fd982144d75923219dca8f35ad1@o1.ingest.sentry.io/2783252>',
```
Comment (benvinegar) [inline src/collections/_documentation/performance/index.md:29]: This looks like a hardcoded DSN and Project got in here? Let's figure out how this is templatized ... or even just put in example values until then.

Comment (PeloWriter) [reply src/collections/_documentation/performance/index.md]: Fixed. Thanks.

File: src/collections/_documentation/performance/getting-started/javascript.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  dsn: '"___PUBLIC_DSN___"',
+  release: 'my-project-name@' + process.env.npm_package_version,
+  integrations: [
+    new ApmIntegrations.Tracing(),
+  ],
+  tracesSampleRate: 0.25, // must be present and non-zero
+});
+```
+
+Performance data is transmitted using a new event type called called "transactions", which you can learn about in [Distributing Tracing]({%- link _documentation/performance/distributed-tracing.md %}#traces-transactions-and-spans). **To sample transactions, you must set the `tracesSampleRate` configuration to a nonzero value.** The example configuration above will transmit 25% of captured transactions.
```
Comment (MimiDumpling) [inline src/collections/_documentation/performance/getting-started/javascript.md:24]:
```suggestion
Performance data is transmitted using a new event type called "transactions", which you can learn about in [Distributing Tracing]({%- link _documentation/performance/distributed-tracing.md %}#traces-transactions-and-spans). **To sample transactions, you must set the `tracesSampleRate` configuration to a nonzero value.** The example configuration above will transmit 25% of captured transactions.
```

File: src/collections/_documentation/performance/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- [Metrics]({% link _documentation/performance/performance-metrics.md %}): provides insight into user experience of your application based on custom thresholds
+- [Discover]({% link _documentation/performance/discover/index.md %}): view data across environments with pre-built queries. Customers subscribed to the Team or Business plan can use Discover to view comprehensive information sent to Sentry.
+
+{% include components/platform_content.html content_dir='getting-started' %}
+
+## Navigating Performance Data
+
+Sentry provides several tools to explore your application's performance data and uncover problems.
+
+The **Performance homepage** is our center for monitoring how your application is doing. Here, you can view a list of all of your endpoints and transactions and view a summary of any transaction in your application. Learn more about the [Homepage]({% link _documentation/performance/performance-homepage.md %}).
```
Comment (MimiDumpling) [inline src/collections/_documentation/performance/index.md:21]:
```suggestion
The **Performance homepage** is our center for monitoring how your application is doing. Here, you can view a list of all your endpoints and transactions and view a summary of any transaction in your application. Learn more about the [Homepage]({% link _documentation/performance/performance-homepage.md %}).
```

Comment (MimiDumpling) [reply src/collections/_documentation/performance/index.md:21]: removing "of" might help the sentence flow more smoothly

File: src/collections/_documentation/performance/getting-started/javascript.md (hunk 2 above)
Comment (MimiDumpling) [reply src/collections/_documentation/performance/getting-started/javascript.md:24]: removing duplicated "called"

Comment (PeloWriter) [reply src/collections/_documentation/performance/getting-started/javascript.md:24]: Fixed. Thanks for the careful read.

File: src/collections/_documentation/performance/index.md (hunk 3 above)
Comment (PeloWriter) [reply src/collections/_documentation/performance/index.md:21]: thank you.

**Pull request merged**